[supervisord]
nodaemon=true

[program:fastapi_service1]
command=python3 /app/data/main.py
autostart=true
autorestart=true
stdout_logfile=/var/log/fastapi_service1.log
stderr_logfile=/var/log/fastapi_service1_err.log

[program:fastapi_service2]
command=python3 main.py
autostart=true
autorestart=true
stdout_logfile=/var/log/fastapi_service2.log
stderr_logfile=/var/log/fastapi_service2_err.log